qt/scientific_interfaces/Muon/ALCLatestFileFinder.cpp

```diff
@@ -25,6 +26,8 @@ std::string ALCLatestFileFinder::getMostRecentFile() const {
     // List all valid Nexus files in the directory
     Poco::Path path(m_firstRunFileName);
     std::vector<std::string> fileNames;
+    // if it found no valid files then return an error
+    // and retry disable the auto button
```

Comment:
It's not important to fix this now, but could this have been worded a bit better.

The behavior described in the error condition is particular unclear (e.g. it mentions a button but this is not a view or presenter).